metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -1452,7 +1485,7 @@ def get_dashboard_description(self, *,
         :return:
         """
 
-        return self._get_resource_description(resource_type=ResourceType.Dashboard, uri=id)
+        return self.get_resource_description(resource_type=ResourceType.Dashboard, uri=id)
 
     @timer_with_counter
     def put_dashboard_description(self, *,
```

Comment:
How about moving calling `put_resource_description` directly from API with the `resource_type=ResourceType.Dashboard`?

Reply:
This would be backwards incompatible so I decided to not do those changes in this PR